docs/dashboards/kubernetes-dashboard.asciidoc

```diff
@@ -0,0 +1,80 @@
+|**GKE**| Regular (default channel): 1.21 and 1.22; Stable: 1.20 and 1.21; Rapid: 1.22 and 1.23 | containerd | Container-optimized OS (COS), Ubuntu
+|=====================
+
+[discrete]
+=== Download and modify the DaemonSet manifest
+The DaemonSet integrates Elastic Endpoint into your Kubernetes cluster. The Elastic Agent is enrolled to a running Fleet Server using the `FLEET_URL` parameter, and connected to a specific Elastic Agent policy using the `FLEET_ENROLLMENT_TOKEN`.
+
+You first need to download the DaemonSet manifest `.yaml`, then modify it to include your {fleet} URL and Enrollment Token before you deploy it to the clusters you want to monitor.
+
+1. Download the DaemonSet manifest http://github.com/elastic/placeholder[here] for GKE or http://github.com/elastic/placeholder[here] for EKS.
+2. Fill in the manifest's `FLEET_URL` field with your {fleet} server's `Host URL`. To find it, go to **{kib} -> Management -> {fleet} -> Settings**. For more information, refer to {fleet-guide}/fleet-settings.html[Fleet UI settings].
+3. Fill in the manifest's `FLEET_ENROLLMENT_TOKEN` field with a Fleet enrollment token. To find one, go to **{kib} -> Management -> {fleet} -> Enrollment tokens**. For more information, refer to {fleet-guide}/fleet-enrollment-tokens.html[Fleet enrollment tokens].
+
+
+[discrete]
+=== Apply the modified manifest to your cluster or clusters
+
+To ensure you install Endpoint on the desired k8s cluster(s), use `kubectl config use-context <name-of-context>` to set the default context. You can repeat the following process for multiple contexts.
+
+NOTE: To check which contexts exist, use `kubectl config get-contexts` to list them from your local kubectl config file. An asterisk, if present, indicates the current default context.
+
+To apply the manifest to a cluster, run for example:
+- EKS: `kubectl apply -f eks-elastic-security.yaml`
+- GKE: `kubectl apply -f gke-elastic-security.yaml`
+
+To check the DaemonSet’s status, run for example: `kubectl get pods -A`
+
+Once the DaemonSet is running, Elastic Endpoint will start sending Linux session data from Kubernetes to {kib}. You can then view that data from the Kubernetes dashboard.
+
+[discrete]
+== The Kubernetes dashboard
+To find the dashboard, go to **Dashboards -> Kubernetes**.
+
+image::kubernetes-dashboard.png[The Kubernetes dashboard, with numbered labels 1 through 3 for major sections]
+The numbered sections are described below:
+
+  1. The charts at the top of the dashboard provide an overview of your monitored Kubernetes infrastructure. You can hide them by clicking *Hide charts*.
+  2. The tree navigation menu allows you to navigate through your deployments and select the scope of the Sessions table to the right. You can select any item in the menu to show its sessions. In Logical view, the menu is organized by Cluster, Namespace, Pod, and Container image. In Infrastructure view, it is organized by Cluster, Node, Pod, and Container image.
```

Comment:
You have "sessions table" lowercase everywhere but in this sentence from what I see.